x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -128,6 +130,24 @@ const ActionsComponent: React.FC<ActionProps> = ({
     }
   }, [dispatch, ecsData._id, timelineId, setGlobalFullScreen, setTimelineFullScreen]);
 
+  const entryLeader = useMemo(() => {
+    const { process } = ecsData;
+    const entryLeaderIds = process?.entry_leader?.entity_id;
+    if (entryLeaderIds !== undefined) {
+      return entryLeaderIds[0];
+    } else {
+      return null;
+    }
+  }, [ecsData]);
```

Comment:
Don't expect `ecsData` to change, but any thoughts on this being `ecsData.process?.entry_leader?.entity_id` to be more specific? Not sure it would make a difference or if the linter would take it, but just a suggestion.

Reply:
Agree, the better is to have a more specific to check.

Reply:
I agree, I changed this when toying with using entry_leader.entity_id at first and then falling back to session_leader/group_leader but that was a bad idea, and I just forgot to change this back ha